bin/pasta_show_maintainers.py

```diff
@@ -0,0 +1,165 @@
+'''
+This work is licensed under the terms of the GNU GPL, version 2.  See
+the COPYING file in the top-level directory.
+'''
+
+import sys
+import os
+
+from csv import writer
+from logging import getLogger
+from collections import OrderedDict
+from argparse import ArgumentParser
+import numpy as np
+import re
+
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
+
+from pypasta.LinuxMaintainers import LinuxMaintainers
+from pypasta.Util import file_to_string
+
+log = getLogger(__name__[-15:])
+
+def flatten(nested):
+    return [item for sublist in nested for item in sublist]
+
+def shorten(string_list):
+    lenght = len(string_list)
+    if lenght>3:
+        return ', '.join(string_list[0:2]) + 'and ' + str(lenght-3) + ' more'
+    elif lenght == 0:
+        return '-'
+    else:
+        return ', '.join(string_list)
+
+def file_len(filename):
+    f = file_to_string(filename)
+    try:
+        return [f.count('\n'), len(f)]
+    # Empty file:
+    except:
+        return [0, 0]
+
+all_filenames = [os.path.join(r, item)[23:] for r, _, f in os.walk('./resources/linux/repo') for item in f]
+
+file_lengths = dict()
+for filename in all_filenames:
+    file_lengths[filename] = file_len('./resources/linux/repo/' + str.strip(filename))
+
+def get_entry_counts(all_maintainers, filenames = all_filenames):
+    subsystem_dirs = dict()
+    counts_by_entry = dict()
+    for filename in filenames:
+        subsystems = all_maintainers.get_subsystems_by_file(filename)
+        for subsystem in subsystems:
+            subsystem_dirs[subsystem] = subsystem_dirs.get(subsystem, set())
+            subsystem_dirs[subsystem].add(''.join(re.findall('[a-zA-Z0-9_]+/', filename)))
+            counts_by_entry[subsystem] = np.add(counts_by_entry.get(subsystem, [0, 0]), file_lengths[filename])
+    return OrderedDict(sorted(counts_by_entry.items(), key=lambda x: x[1][1], reverse=True)), subsystem_dirs
+
+def get_maintainer_counts(all_maintainers, filenames = all_filenames):
+    maintainers_subs = dict()
+    counts_by_maintainer = dict()
+    for filename in filenames:
+        subsystems = all_maintainers.get_subsystems_by_file(filename)
+        for subsystem in subsystems:
+            maintainers = all_maintainers.get_maintainers(subsystem)
+            for maintainer in flatten(maintainers):
+                if not (type(maintainer) == str or maintainer[0] == '' or maintainer[1] == ''):
+                    maintainers_subs[maintainer[0]+' <'+maintainer[1]+'>'] = maintainers_subs.get(maintainer[0]+' <'+maintainer[1]+'>', set())
```

Comment:
maintainer[0]+' <'+maintainer[1]+'>' is a pretty-printing function or a to_string, or string_of_maintainer function.